backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/NormService.java

```diff
@@ -136,24 +141,36 @@ public String updateRegelungstextXml(UpdateRegelungstextXmlUseCase.Options optio
     return XmlMapper.toString(updatedRegelungstext.getDocument());
   }
 
+  private void prepareNormForSaving(Norm norm) {
+    norm
+      .getDokumente()
+      .forEach(dokument -> {
+        EidConsistencyGuardian.eliminateDeadReferences(dokument.getDocument());
+        EidConsistencyGuardian.correctEids(dokument.getDocument());
+        ldmlDeElementSorter.sortElements(dokument.getDocument().getDocumentElement());
+      });
+
+    ldmlDeValidator.validateXSDSchema(norm);
+  }
+
   /**
-   * It not only saves a {@link Norm} but makes sure that all Eids are consistent.
+   * It not only saves a {@link Norm} but makes sure that all Eids are consistent. It always saves to the working copy
```

Comment:
Would adapt the docs so that it does not only talk about correcting the Eids but rather doing all the pre-processing needed (maybe just linking to the private method or better add all the pre-processing steps here?)